src/Concordium/Client/Commands.hs

```diff
@@ -1767,6 +1773,25 @@ bakerUpdateOpenDelegationStatus =
             (progDesc "Change whether to allow other parties to delegate stake to the baker.")
         )
 
+bakerGetEarliestWinTime :: Mod CommandFields BakerCmd
+bakerGetEarliestWinTime =
+    command
+        "win-time"
+        ( info
+            ( BakerGetEarliestWinTime
+                <$> argument auto (metavar "BAKER-ID" <> help "Baker ID to query.")
+                <*> switch
+                    ( long "local-time"
+                        <> help "Display time in the local time zone (instead of UTC)."
+                    )
+                <*> switch
+                    ( long "poll"
+                        <> help "Repeatedly poll for the latest time."
```

Comment:
What is the polling interval? Is it relevant for the user? Should it be mentioned?

Reply:
Should it be configurable?

Reply:
I think the way it works now does not need any configuration for the polling interval.

Reply:
The interval is variable: 10 seconds if the last reported time was more than 10 seconds in the future, and 1 second otherwise. I don't think there's a need to communicate this or configure it, as this should be fine for most purposes.